sql/core/src/test/resources/sql-tests/inputs/datetime-corrected.sql

```diff
@@ -0,0 +1,2 @@
+--SET spark.sql.legacy.timeParserPolicy=CORRECTED
```

Comment:
do we have different test results with CORRECTED mode?

Reply:
I've just come to understand what it means, I will rm this case.